Okta 465848 465914 466346 oie intro page, banners, limited GA

chrisdmills-okta opened the pull request:
<!-- PLEASE REMEMBER THAT THIS IS A PUBLIC REPO AND ANY PR AND SUBSEQUENT DISCUSSION WILL BE VISIBLE TO ANYONE AND EVERYONE -->

## Description:
- **What's changed?** This PR handles a number of changes we are looking at implementing to go along with the OIE-by-default docs work:
  - Updating the "Identity Engine" banners at the top of OIE feature pages so that they link to our [OIE intro page](https://developer.okta.com/docs/guides/oie-intro/), rather than the Okta help center OIE page.
  - Updating said intro page so that it does a better job of providing an actual intro to OIE — what it is, what new features it has, what fundamental differences it has to classic, how to get started with implementing the different auth models in OIE, how we've changed out docs to support OIE, and how to actually make use of it or upgrade to it if you are still on classic.
  - Removing the "Limited GA" banners from OIE pages and features — once the launch has happened, it will no longer be limited GA. I also took this opportunity to remove some non-standard styling divs that were used during the tuscany docs project; I think their need should be talked about and handled in a standard site-wide way, if we feel it is necessary.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-465848](https://oktainc.atlassian.net/browse/OKTA-465848)
* [OKTA-465914](https://oktainc.atlassian.net/browse/OKTA-465914)
* [OKTA-466346](https://oktainc.atlassian.net/browse/OKTA-466346)

chrisdmills-okta commented:
For reviewers:

* You can see a page with the "Identity Engine" banner here: https://61fbc12ffd41ca7918fc9459--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-org-setup/android/main/. This will click through to the below page
* You can see the Indentity Engine intro page here: https://61fbc12ffd41ca7918fc9459--reverent-murdock-829d24.netlify.app/docs/guides/oie-intro/. I'd love to know what you think about:
  * The content, and structure of the page
  * Whether the voice/tone is appropriate (remember it's for a developer audience)
  * What else should be covered here (you'll note that the first two sections have some ALL CAPS COMMENTS asking about what else should be called out)

susanharper-okta commented:
Here is my two cents:

**App-level policies**
Should mention the ability to share policies across applications. And there will be a link that you can include to our content, also - the Configure Global Session Policy and app sign-on policies guide. Also the Policies concept piece as it is also being updated for OIE with the shareable app effort that I'm currently working on.

**Policy-driven** 
Should link to the Configure Global Session Policy and app sign-on policies guide and the Policies concept piece as it is also being updated for OIE with the shareable app effort that I'm currently working on.

**Redirect authentication**
Isn't my redirect auth guide being split up into the sign users in guides? Just making note to change the link to the Sign Users In landing page? Is that how that will work?

Also - seems confusing to direct the reader to the embedded page for setting up the OIE sample apps when the redirect auth doc also discusses setting up the sample apps for use with the guide?

**Identity Engine versus Classic documentation approach**
The wording of "first-class citizen" sort of provides a negative feel, IMHO. Might want to avoid that type of reference?

**Accessing and upgrading to Identity Engine**
The "For classic customers who don't yet want to upgrade" section - is this designed to provide them with a bulleted list of reasons that to upgrade? If so, that's not super clear....as the first bullet seems to be about how the APIs still work as before if they don't upgrade...but the second two bullets seem to focus on "your stuff will still work if you upgrade"...gives two separate feels to the bulleted list....unless I'm not reading this correctly.

chrisdmills-okta commented:
> Here is my two cents:

Great, thanks @susanharper-okta !
 
> **App-level policies** Should mention the ability to share policies across applications. And there will be a link that you can include to our content, also - the Configure Global Session Policy and app sign-on policies guide. Also the Policies concept piece as it is also being updated for OIE with the shareable app effort that I'm currently working on.

All sounds good; updated in my latest commit. I don't think these changes will be in the Netlify deploy; I'd have to trigger a new build manually, correct?

> **Policy-driven** Should link to the Configure Global Session Policy and app sign-on policies guide and the Policies concept piece as it is also being updated for OIE with the shareable app effort that I'm currently working on.

Again, added in latest commit

> **Redirect authentication** Isn't my redirect auth guide being split up into the sign users in guides? Just making note to change the link to the Sign Users In landing page? Is that how that will work?

So, we're not splitting up your redirect auth article. We are keeping it mostly intact (although I think it will require a bit of updating to point to the other guides, show the relationship there), alongside the new redirect auth sign in guides. The new guides provide the basicas of set up and signing in your first user, and your guide then shows some more advanbced use cases, and how to use redirect auth with the IDX sample apps. We are using simpler sample apps in the former, so it is easy for the reader to rapidly get what is going on.
 
> Also - seems confusing to direct the reader to the embedded page for setting up the OIE sample apps when the redirect auth doc also discusses setting up the sample apps for use with the guide?

This is a good point. I've removed that link.

> 
> **Identity Engine versus Classic documentation approach** The wording of "first-class citizen" sort of provides a negative feel, IMHO. Might want to avoid that type of reference?

Agreed; I've changed it.

> 
> **Accessing and upgrading to Identity Engine** The "For classic customers who don't yet want to upgrade" section - is this designed to provide them with a bulleted list of reasons that to upgrade? If so, that's not super clear....as the first bullet seems to be about how the APIs still work as before if they don't upgrade...but the second two bullets seem to focus on "your stuff will still work if you upgrade"...gives two separate feels to the bulleted list....unless I'm not reading this correctly.

This is a good point. I've changed it so that the bullets relevant to upgrading are in a bulleted list, and the bit about what will still work if you don't upgrade is handled separately.
